Sources/TuistKit/Generator/Generator.swift

```diff
@@ -196,30 +74,12 @@ class Generator: Generating {
         return (workspaceDescriptor.xcworkspacePath, graph)
     }
 
-    internal func generateProjectWorkspace(path: AbsolutePath) async throws -> (AbsolutePath, Graph) {
-        // Load
-        let (_, graph, sideEffects) = try await loadProjectWorkspace(path: path)
-        let graphTraverser = GraphTraverser(graph: graph)
-
-        // Lint
-        try lint(graphTraverser: graphTraverser)
-
-        // Generate
-        let workspaceDescriptor = try generator.generateWorkspace(graphTraverser: graphTraverser)
-
-        // Write
-        try writer.write(workspace: workspaceDescriptor)
-
-        // Mapper side effects
-        try sideEffectDescriptorExecutor.execute(sideEffects: sideEffects)
-
-        // Post Generate Actions
-        try await postGenerationActions(
-            graphTraverser: graphTraverser,
-            workspaceName: workspaceDescriptor.xcworkspacePath.basename
-        )
+    func load(path: AbsolutePath) async throws -> Graph {
+        try await load(path: path).0
+    }
 
-        return (workspaceDescriptor.xcworkspacePath, graph)
+    func load(path: AbsolutePath) async throws -> (Graph, [SideEffectDescriptor]) {
+        try await manifestGraphLoader.load(path: path)
     }
```

Comment:
Is it used only internally? can we remove it and just call manifestGraphLoader above?